compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -778,7 +779,7 @@ convertBind env (name, x)
 -- during conversion to DAML-LF together with their constructors since we
 -- deliberately remove 'GHC.Types.Opaque' as well.
 internalTypes :: UniqSet FastString
-internalTypes = mkUniqSet ["Scenario","Update","ContractId","Time","Date","Party","Pair", "TextMap", "Map", "Any", "TypeRep"]
+internalTypes = mkUniqSet ["Scenario","Update","ContractId","Time","Date","Party","Pair", "TextMap", "Map", "Any", "TypeRep", "Optional"]
```

Comment:
Do we really need to add `Optional` to that list? It's a list of types defined in `DA.Internal.LF` whereas `Optional` is defined in `DA.Internal.Prelude`.

Reply:
Good catch, that was leftover from when I tried to move it to `DA.Internal.LF`. Fixed in https://github.com/digital-asset/daml/pull/3981